go/interpreter/lfvm/instructions_test.go

```diff
@@ -880,3 +880,41 @@ func TestExpansionCostOverflow(t *testing.T) {
 		}
 	}
 }
+
+func TestGetAccessCost_RespondsWithProperGasPrice(t *testing.T) {
+	if want, got := tosca.Gas(100), getAccessCost(tosca.WarmAccess); want != got {
+		t.Errorf("unexpected gas cost, wanted %d, got %d", want, got)
+	}
+	if want, got := tosca.Gas(2600), getAccessCost(tosca.ColdAccess); want != got {
+		t.Errorf("unexpected gas cost, wanted %d, got %d", want, got)
+	}
+}
+
+func TestCall_ChargesNothingForColdAccessBeforeBerlin(t *testing.T) {
```

Comment:
This test should have a counter-part checking that starting with Berlin gas costs for cold/warm accesses are charged.